controller/label_blackbox_test.go

```diff
@@ -42,6 +42,25 @@ func (rest *TestLabelREST) SetupTest() {
 	rest.testDir = filepath.Join("test-files", "label")
 }
 
+func (rest *TestLabelREST) TestCreateLabelNoSpace() {
+	fxt, err := tf.NewFixture(rest.DB, tf.Identities(1), tf.Spaces(1))
+	require.NoError(rest.T(), err)
+	svc := testsupport.ServiceAsUser("Label-Service", *fxt.Identities[0])
+	ctrl := NewLabelController(svc, rest.db, rest.Configuration)
+	color := "some color"
+	pl := app.CreateLabelPayload{
+		Data: &app.Label{
+			Attributes: &app.LabelAttributes{Name: &color},
+			Type:       label.APIStringTypeLabels,
+		},
+	}
+	_, jerrs := test.CreateLabelNotFound(rest.T(), svc.Context, svc, ctrl, uuid.Nil, &pl)
+	require.NotNil(rest.T(), jerrs)
+	require.Len(rest.T(), jerrs.Errors, 1)
+	require.Contains(rest.T(), jerrs.Errors[0].Detail, "spaces with id '00000000-0000-0000-0000-000000000000' not found")
```

Comment:
I think you can use golden file here for both payload & error response. See `controller/iteration_blackbox_test.go` for an example.

Reply:
@baijum yes I will use golden file

Reply:
@baijum added golden file. See 5999c4d and ea8867c

Reply:
You can create another golden file for request payload.

Reply:
@baijum will add another golden file for request payload

Reply:
@baijum added golden file for request payload and header -> 429f03f